esmvalcore/preprocessor/_other.py

```diff
@@ -38,3 +39,93 @@ def clip(cube, minimum=None, maximum=None):
             raise ValueError("Maximum should be equal or larger than minimum.")
     cube.data = da.clip(cube.core_data(), minimum, maximum)
     return cube
+
+
+def add_cell_measure(cube, fx_variables, check_level):
+    """
+    Load requested fx files, check with CMOR standards and add the
+    fx variables as cell measures in the cube containing the data.
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        Iris cube with input data.
+    fx_variables: dict
+        Dictionary with fx_variable information.
+    check_level: CheckLevels
+        Level of strictness of the checks.
+
+
+    Returns
+    -------
+    iris.cube.Cube
+        Cube with added cell measures.
+    """
+    from esmvalcore.preprocessor._io import concatenate
+    from esmvalcore.cmor.fix import fix_metadata, fix_data
+    from esmvalcore.cmor.check import cmor_check_metadata, cmor_check_data
+
+    if not fx_variables:
+        return cube
+    fx_cubes = iris.cube.CubeList()
+    for _, fx_info in fx_variables.items():
+        if not fx_info:
+            continue
+        if isinstance(fx_info['filename'], str):
+            fx_info['filename'] = [fx_info['filename']]
+        for fx_file in fx_info['filename']:
+            loaded_cube = iris.load(fx_file)
+            short_name = fx_info['short_name']
+            project = fx_info['project']
+            dataset = fx_info['dataset']
+            mip = fx_info['mip']
+            freq = fx_info['frequency']
+            loaded_cube = fix_metadata(loaded_cube, short_name=short_name,
+                                       project=project, dataset=dataset,
+                                       mip=mip, frequency=freq,
+                                       check_level=check_level)
+            fx_cubes.append(loaded_cube[0])
+
+        fx_cube = concatenate(fx_cubes)
+
+        fx_cube = cmor_check_metadata(fx_cube, cmor_table=project, mip=mip,
+                                      short_name=short_name, frequency=freq,
+                                      check_level=check_level)
+
+        fx_cube = fix_data(fx_cube, short_name=short_name, project=project,
+                           dataset=dataset, mip=mip, frequency=freq,
+                           check_level=check_level)
+
+        fx_cube = cmor_check_data(fx_cube, cmor_table=project, mip=mip,
+                                  short_name=fx_cube.var_name, frequency=freq,
+                                  check_level=check_level)
+
+        measure_name = {
+            'areacella': 'area',
+            'areacello': 'area',
+            'volcello': 'volume'
+        }
+
+        if fx_cube.var_name in measure_name.keys():
```

Comment:
```suggestion
        if fx_cube.var_name in measure_name:
```
there is no need to use the `.keys()` method, `in` will look at the keys in a dict by default